include/fizzy/fizzy.h

```diff
@@ -6,6 +6,7 @@
 /// @file
 #pragma once
 
+#include "fizzy_errors.h"
```

Comment:
Making this `<fizzy/fizzy_errors.h>` breaks the Rust build, not sure yet how to fix.

Reply:
Why do we want to separate it out? Also what is the error Rust gives?

Reply:
I wanted to separate it because it is used in `lib/fizzy/exceptions.hpp`.

But now I'm thinking it might be not needed for the fist version, if we just make our 3 exception classes mean 3 possible error codes.

Reply:
Error was like this:
```
 --- stderr
  fizzy/include/fizzy/fizzy.h:10:10: fatal error: 'fizzy/fizzy_errors.h' file not found
  fizzy/include/fizzy/fizzy.h:10:10: fatal error: 'fizzy/fizzy_errors.h' file not found, err: true
  thread 'main' panicked at 'Unable to generate bindings: ()', bindings/rust/build.rs:33:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
Probably need to add some include path into `build.rs`.